vaadin-archetype-application/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -0,0 +1,12 @@
+dir = new File(new File(request.outputDirectory), request.artifactId)
```

Comment:
If we plan to cut versions of archetypes with all platform releases, we can remove this file. If not, this part needs some more testing and should probably be disabled in certain environments where it fails.

We can probably do the testing with viritin archetypes if needed, but I think some Windows user reported a problem and from Eclipse at least the script isn't run at all.

Reply:
Done

Reply:
I meant the whole archetype-post-generate.groovy file. Now the automatic upgrade to latest version probably don't work from CLI either.